integrate/mcp.md

```diff
@@ -0,0 +1,197 @@
+        "keboola": {
+          "command": "docker",
+          "args": [
+            "run",
+            "--it", 
+            "--rm", 
+            "-e", "KBC_STORAGE_TOKEN", 
+            "-e", "KBC_WORKSPACE_SCHEMA", 
+            "keboola/mcp-server:latest", 
+            "--api-url", "https://connection.YOUR_REGION.keboola.com"
+          ],
+          "env": {
+            "KBC_STORAGE_TOKEN": "YOUR_KEBOOLA_STORAGE_TOKEN",
+            "KBC_WORKSPACE_SCHEMA": "YOUR_KEBOOLA_STORAGE_TOKEN"
+          }
+        }
+      }
+    }
+    ```
+    **Note:**
+    *   Ensure Docker is running on your system.
+    *   Replace placeholders like `YOUR_KEBOOLA_STORAGE_TOKEN`, `YOUR_WORKSPACE_SCHEMA`, and the Keboola API URL.
+    *   The client (Cursor) passes the `KBC_STORAGE_TOKEN` and `KBC_WORKSPACE_SCHEMA` from its `env` block to the `docker run` command through the `-e` flags. The `--api-url` is passed directly as an argument to the `keboola/mcp-server` entrypoint.
+
+## Running Keboola MCP Server Locally Using Python
+
+While MCP clients like Cursor or Claude typically manage the MCP server automatically, you might want to run the Keboola MCP Server locally for development, testing, or when using a custom client.
+
+The primary way to run the server locally is by cloning the [Keboola MCP Server GitHub repository](https://github.com/keboola/mcp-server) and running it using Python. Make sure you have Python 3.10+ and `uv` installed.
+
+1.  **Clone the repository:**
+    ```bash
+    git clone https://github.com/keboola/mcp-server.git
+    cd mcp-server
+    ```
+2.  **Install dependencies (including for development/testing if needed)**:
+    ```bash
+    uv sync --extra dev 
+    ```
+3.  **Set up environment variables:**
+    Before running the server, you need to configure the following environment variables:
+    *   `KBC_STORAGE_TOKEN`: Your Keboola Storage API token.
+    *   `KBC_WORKSPACE_SCHEMA`: Your Keboola project's workspace schema (for SQL queries).
+    *   `KBC_API_URL`: Your Keboola instance API URL (e.g., `https://connection.keboola.com` or `https://connection.YOUR_REGION.keboola.com`).
+
+    Refer to the [Keboola Tokens](https://help.keboola.com/management/project/tokens/) and [Keboola workspaces manipulation](https://help.keboola.com/tutorial/manipulate/workspace/) for detailed instructions on obtaining these values.
+
+    **3.1. Additional Setup for BigQuery Users**
+
+    If your Keboola project uses BigQuery as its backend, you will also need to set up the `GOOGLE_APPLICATION_CREDENTIALS` environment variable. This variable should point to the JSON file containing your Google Cloud service account key that has the necessary permissions to access your BigQuery data.
+
+    Example:
+    `GOOGLE_APPLICATION_CREDENTIALS="/path/to/your/credentials.json"`
+
+4.  **Run the server (example using Python module directly for testing CLI manually)**:
+    ```bash
+    uv run python -m src.keboola_mcp_server.cli --api-url $KBC_API_URL
```

Comment:
@Matovidlo `src` is not a python module, just a source folder the command should be `uv run python -m keboola_mcp_server`